prodtests/full-system-test/dpl-workflow.sh

```diff
@@ -202,7 +216,13 @@ for det in `echo $LIST_OF_DETECTORS | sed "s/,/ /g"`; do
     has_detector_reco $det && add_comma_separated TRACK_SOURCES "$det"
   fi
 done
-PVERTEX_CONFIG="--vertexing-sources $TRACK_SOURCES --vertex-track-matching-sources $TRACK_SOURCES"
+
+if [ -z $ISPILOTBEAMDATA ]; then
+  PVERTEX_CONFIG="--vertexing-sources $TRACK_SOURCES --vertex-track-matching-sources $TRACK_SOURCES"
+else
+  PVERTEX_CONFIG="--vertexing-sources ITS --vertex-track-matching-sources ITS --hbfutils-config none"
+  PVERTEX_EXTRA_CONFIG="pvertexer.nSigmaTimeCut=100;pvertexer.dbscanMaxDist2=30;pvertexer.dcaTolerance=3.;pvertexer.pullIniCut=100;pvertexer.addZSigma2=0.1;pvertexer.tukey=20.;pvertexer.addZSigma2Debris=0.01;pvertexer.addTimeSigma2Debris=1.;pvertexer.maxChi2Mean=30;"
```

Comment:
Please use `CONFIG_EXTRA_PROCESS_o2_primary_vertexing_workflow="..."`, the other is deprecated and will be removed.

Reply:
Done.